alf/experience_replayers/replay_buffer.py

```diff
@@ -27,13 +27,19 @@
 from .segment_tree import SumSegmentTree, MaxSegmentTree, MinSegmentTree
 
 BatchInfo = namedtuple(
-    "BatchInfo", ["env_ids", "positions", "importance_weights"],
-    default_value=())
+    "BatchInfo", ["env_ids", "index", "importance_weights"], default_value=())
```

Comment:
We can still use positions so that we can know whether a position is outdated (removed) during update_priority()